languages/elixir/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,112 @@
+In this exercise you're going to write a module to create and manipulate a map of high scores.
```

Comment:
I like this story! Maybe we can leave out the "module" bit (that is an implementation detail, not part of the story) and expand slightly on the story to give it a bit of a flourish?

```suggestion
In this exercise, you're implementing a way to keep track of the high scores for the most popular game in your local arcade hall.
```

Or something like that.